__tests__/app.test.js

```diff
@@ -96,3 +96,50 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("should respond with an array of articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const body = response.body;
+
+        expect(body.articles.length).toBe(13);
```

Comment:
Well done asserting the number of articles we expect to get back, as well as checking for each property to be the correct type